### DISCUSSION TITLE
front-end and back-end separation deployment？

### DISCUSSION THREAD

**Discussion by zhangxinyao521** [apache/skywalking#10897] [General]
can the front-end and back-end be deployed separately?

**Comment by wu-sheng** [comment]
What is the frontend and backend? What are you talking about?

**Reply by zhangxinyao521** [reply]
I encountered a compilation issue during deployment,skywalking-ui build fail.Is it okay not to compile skywalking?

**Reply by wu-sheng** [reply]
I don't know what you expect something in the official repo could be excluded. This is like a joke. I don't have anything valuable to say.
Strange.
